Libplanet/Net/Swarm.cs

```diff
@@ -231,21 +243,19 @@ private set
                 timestamp = DateTimeOffset.UtcNow;
             }
 
-            foreach (Peer peer in peers)
+            var peersAsArray = peers as Peer[] ?? peers.ToArray();
+            foreach (Peer peer in peersAsArray)
             {
                 if (_removedPeers.ContainsKey(peer))
                 {
                     _removedPeers.Remove(peer);
                 }
             }
 
-            var existingKeys = new HashSet<PublicKey>(
-                _peers.Keys.Select(p => p.PublicKey)
-            );
```

Comment:
Has this been unnecessary?

Reply:
Yes, it's unnecessary because there is no usage in this scope.